package.json

```diff
@@ -31,9 +30,12 @@
     "uglify-js": "^2.4.16",
     "watchify": "^2.2.1"
   },
+  "peerDependencies": {
+    "coffeeify": "^1.0.0"
```

Comment:
I think the browserify transform in package.json is causing it to look for it. Running `npm start` is throwing `Error: Cannot find module 'coffeeify'`